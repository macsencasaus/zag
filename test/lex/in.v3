// RUN: %vector -lex %s

x: i32;

fn main(argc: i32, argv: **i8): i32 {
    var x = 0;
    return x;
}
