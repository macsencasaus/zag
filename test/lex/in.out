var: var at 3:1 in lex/in.v3
identifier: x at 3:5 in lex/in.v3
':': : at 3:6 in lex/in.v3
identifier: i32 at 3:8 in lex/in.v3
';': ; at 3:11 in lex/in.v3
fn: fn at 5:1 in lex/in.v3
identifier: main at 5:4 in lex/in.v3
'(': ( at 5:8 in lex/in.v3
identifier: argc at 5:9 in lex/in.v3
':': : at 5:13 in lex/in.v3
identifier: i32 at 5:15 in lex/in.v3
',': , at 5:18 in lex/in.v3
identifier: argv at 5:20 in lex/in.v3
':': : at 5:24 in lex/in.v3
'*': * at 5:26 in lex/in.v3
'*': * at 5:27 in lex/in.v3
identifier: i8 at 5:28 in lex/in.v3
')': ) at 5:30 in lex/in.v3
':': : at 5:31 in lex/in.v3
identifier: i32 at 5:33 in lex/in.v3
'{': { at 5:37 in lex/in.v3
var: var at 6:5 in lex/in.v3
identifier: x at 6:9 in lex/in.v3
'=': = at 6:11 in lex/in.v3
int literal: 0 at 6:13 in lex/in.v3
';': ; at 6:14 in lex/in.v3
var: var at 7:5 in lex/in.v3
identifier: y at 7:9 in lex/in.v3
'=': = at 7:11 in lex/in.v3
identifier: x at 7:13 in lex/in.v3
'+': + at 7:15 in lex/in.v3
identifier: x at 7:17 in lex/in.v3
';': ; at 7:18 in lex/in.v3
identifier: x at 8:5 in lex/in.v3
'-': - at 8:7 in lex/in.v3
identifier: y at 8:9 in lex/in.v3
';': ; at 8:10 in lex/in.v3
identifier: x at 9:5 in lex/in.v3
'*': * at 9:7 in lex/in.v3
identifier: y at 9:9 in lex/in.v3
';': ; at 9:10 in lex/in.v3
identifier: x at 10:5 in lex/in.v3
'/': / at 10:7 in lex/in.v3
identifier: y at 10:9 in lex/in.v3
';': ; at 10:10 in lex/in.v3
identifier: x at 11:5 in lex/in.v3
'^': ^ at 11:7 in lex/in.v3
identifier: y at 11:9 in lex/in.v3
';': ; at 11:10 in lex/in.v3
identifier: x at 12:5 in lex/in.v3
'&': & at 12:7 in lex/in.v3
identifier: y at 12:9 in lex/in.v3
';': ; at 12:10 in lex/in.v3
identifier: x at 13:5 in lex/in.v3
'|': | at 13:7 in lex/in.v3
identifier: y at 13:9 in lex/in.v3
';': ; at 13:10 in lex/in.v3
'~': ~ at 14:5 in lex/in.v3
identifier: x at 14:6 in lex/in.v3
';': ; at 14:7 in lex/in.v3
'!': ! at 15:5 in lex/in.v3
identifier: x at 15:6 in lex/in.v3
';': ; at 15:7 in lex/in.v3
'++': ++ at 16:5 in lex/in.v3
identifier: x at 16:7 in lex/in.v3
';': ; at 16:8 in lex/in.v3
identifier: x at 17:5 in lex/in.v3
'++': ++ at 17:6 in lex/in.v3
';': ; at 17:8 in lex/in.v3
'--': -- at 18:5 in lex/in.v3
identifier: x at 18:7 in lex/in.v3
';': ; at 18:8 in lex/in.v3
identifier: x at 19:5 in lex/in.v3
'--': -- at 19:6 in lex/in.v3
';': ; at 19:8 in lex/in.v3
identifier: x at 20:5 in lex/in.v3
'<=': <= at 20:7 in lex/in.v3
identifier: y at 20:10 in lex/in.v3
';': ; at 20:11 in lex/in.v3
identifier: x at 21:5 in lex/in.v3
'<': < at 21:7 in lex/in.v3
identifier: y at 21:9 in lex/in.v3
';': ; at 21:10 in lex/in.v3
identifier: x at 22:5 in lex/in.v3
'==': == at 22:7 in lex/in.v3
identifier: y at 22:10 in lex/in.v3
';': ; at 22:11 in lex/in.v3
identifier: x at 23:5 in lex/in.v3
'!=': != at 23:7 in lex/in.v3
identifier: x at 23:10 in lex/in.v3
';': ; at 23:11 in lex/in.v3
identifier: x at 24:5 in lex/in.v3
'>': > at 24:7 in lex/in.v3
identifier: y at 24:9 in lex/in.v3
';': ; at 24:10 in lex/in.v3
identifier: x at 25:5 in lex/in.v3
'>=': >= at 25:7 in lex/in.v3
identifier: y at 25:10 in lex/in.v3
';': ; at 25:11 in lex/in.v3
return: return at 26:5 in lex/in.v3
identifier: x at 26:12 in lex/in.v3
';': ; at 26:13 in lex/in.v3
'}': } at 27:1 in lex/in.v3
EOF:  at 28:1 in lex/in.v3
